### PR TITLE
Update to Django 1.7

### PR DESCRIPTION
**Pull request by Hwesta**
Update Archivematica to use Django 1.7.  Several changes made to update backwards incompatible changes, notably that ModelForms must specify their fields, and that standalone scripts (all the clientScripts) must setup Django before calling it.

### CODE REVIEW

**Comment by mistydemeo** [comment]
:+1:

**Comment by Hwesta** [comment]
In progress updating for #98 changes

File: src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py [hunk 1]
```diff
@@ -8,18 +8,14 @@
 from tempfile import gettempdir
 from uuid import uuid4
 
-path = '/usr/share/archivematica/dashboard'
-if path not in sys.path:
-    sys.path.append(path)
+import django
+sys.path.append('/usr/share/archivematica/dashboard')
```
Comment (mistydemeo) [inline src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py]: I wonder if it's worth changing MCPClient to export `PYTHONPATH` and `DJANGO_SETTINGS_MODULE` when spinning up subprocesses? That way we could avoid requiring every client script to set its own `sys.path`.

Comment (mistydemeo) [inline src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py]: (Not that that needs to be in this PR, it just occurred to me.)

File: src/dashboard/src/components/rights/forms.py [hunk 2]
```diff
@@ -81,12 +79,14 @@ class Meta:
 class RightsCopyrightNoteForm(ModelForm):
     class Meta:
         model = models.RightsStatementCopyrightNote
+        fields = ('copyrightnote',)  # TODO check this is correct
```
Comment (mistydemeo) [inline src/dashboard/src/components/rights/forms.py]: Yep, `copyrightNote` is the only non-pk/non-fk field

File: src/dashboard/src/external/archivematica-fpr-admin [hunk 3]
```diff
@@ -1 +1 @@
-Subproject commit c253d9e447344c3369c581025acaf016d8e114d4
+Subproject commit e87f9196d70eef1f5c1525c851859dd8bcd66568
```
Comment (mistydemeo) [inline src/dashboard/src/external/archivematica-fpr-admin]: Was this intentional?

**Comment by mistydemeo** [comment]
Still :+1:

File: src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py (hunk 1 above)
Comment (Hwesta) [inline src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py]: That is a great idea, looking into it.  We use threads not subprocesses currently, so I dunno how different it'll be...

Comment (mistydemeo) [inline src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py]: We still run them as separate CLI scripts, so they're definitely being spun off as new processes at some level. IIRC our subprocess wrapper does support exporting the environment too.

File: src/dashboard/src/external/archivematica-fpr-admin (hunk 3 above)
Comment (Hwesta) [inline src/dashboard/src/external/archivematica-fpr-admin]: Yes - updates to match artefactual/archivematica-fpr-admin#8

**Comment by Hwesta** [comment]
Includes #161

**Comment by mistydemeo** [comment]
Still :+1:

File: src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py (hunk 1 above)
Comment (Hwesta) [inline src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py]: This was fixed in #161 (after I forgot this conversation and did the work anyway)

**Comment by Hwesta** [comment]
Added updates to postinst so the package upgrade should work, but needs to be tested.  Updated one of the databaseFunction tests, because updating to 1.7 broke it somehow. Responded to feedback.

**Comment by mistydemeo** [comment]
:shipit:

**Pull request merged**